plugins/ssr/ssrvernac.mlg

```diff
@@ -300,6 +300,33 @@ VERNAC COMMAND EXTEND HintView CLASSIFIED AS SIDEFF
           Ssrview.AdaptorDb.declare k hints }
 END
 
+(** Search compatibility *)
+
+{
+  let warn_search_moved = CWarnings.create ~name:"ssr-search-moved"
+      ~category:"deprecation" ~default:CWarnings.Enabled
+      (fun () ->
+         (Pp.strbrk
+            "ssreflect Search command has been moved to the \
+             ssrsearch module; please Require that module if you \
+             still want to use ssreflect's Search command"))
```

Comment:
Use official casing:

```suggestion
            "SSReflect's Search command has been moved to the \
             ssrsearch module; please Require that module if you \
             still want to use SSReflect's Search command"))
```

Reply:
I think the message should invite the user to try out the "vanilla" search and only require ssrsearch after filing an issue (of a missing feature for example).

The message assumes the user knows why the command was moved, which may not be the case.

Reply:
Note that the main reason for moving the command is because it overwrites the default one (see also #12253). The deprecation message emitted by `Search` in `ssrsearch` encourages the use of the default `Search` with the `headconcl:` clause instead.

Reply:
Oh, I see, the warning is implemented in ssr/ssrvernac.mlg but is only issues by ssrsearch/g_ssrsearch.mlg, am I right?

Reply:
There are two warnings:
- the one in ssrvernac is only emitted if the ssrsearch plugin is not loaded, and it tells the user that this is not anymore the SSReflect's `Search`
- the one in ssrsearch is emitted when using SSReflect's `Search` and recommends using the `headconcl:` clause of the normal `Search`.

Does that make sense? Would you do it differently?

Reply:
I have to try it out, I guess. I will before the final